public/index.html

```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
```

Comment:
DNPTL: On further thought, getting rid of the public folder completely was probably not the move 😂

Reply:
Yeah, you still need the root `div` for React to mount to something. 🙂